test/unit/L2/delegatorPool.test.ts

```diff
@@ -176,10 +184,39 @@ describe('DelegatorPool', function() {
               .claim(delegator.address, stake);
 
           await expect(tx).to.be.revertedWith(
-              'DelegatorPool#claim: FULLY_CLAIMED',
+              'DelegatorPool#claim: INVALID_CLAIM',
           );
         });
 
+        it('fails if requested stake exceeds initial stake', async () => {
+          const unclaimedTokens = 10;
+          await delegatorPool
```

Comment:
I suggest adding the following expect line before these test cases:

```
expect(await delegatorPool.initialStake()).to.be.equal(stake)
```

in order to convey to the reader that `stake` is equal to the `initialStake` tracked by the DelegatorPool so that in the second test case below it is clear why claiming `unclaimedTokens + 1` would trigger a revert (i.e. since the first claim was for `initialStake - unclaimedTokens`).

Reply:
addressed in b2b88fe